docs/user/build-customization.rst

```diff
@@ -344,30 +344,53 @@ Override the build process
 
 .. warning::
 
-   This feature is in a *beta phase* and could suffer incompatible changes or even removed completely in the near feature.
-   It does not yet support some of the Read the Docs' features like the :term:`flyout menu`, and ads.
-   However, we do plan to support these features in the future.
-   Use this feature at your own risk.
-
+   This feature is in *beta* and could change without warning.
+   It does not yet support some of the Read the Docs' features like the :term:`flyout menu`.
+   We do our best to not break existing configurations,
+   but use this feature at your own risk.
 
 If your project requires full control of the build process,
 and :ref:`extending the build process <build-customization:extend the build process>` is not enough,
-all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands` configuration file key.
+all the commands executed during builds can be overridden using the :ref:`config-file/v2:build.commands`.
 
 As Read the Docs does not have control over the build process,
-you are responsible for running all the commands required to install requirements and build your project properly.
-Once the build process finishes, the contents of the ``_readthedocs/html/`` directory will be hosted.
+you are responsible for running all the commands required to install requirements and build your project.
+
+Where to put files
+~~~~~~~~~~~~~~~~~~
 
+It is your responsibility to generate HTML and other formats of your documentation using :ref:`config-file/v2:build.commands`.
+The contents of the ``_readthedocs/<format>/`` directory will be hosted as part of your documentation.
```

Comment:
```suggestion
The contents of the ``$READTHEDOCS_OUTPUT/<format>/`` directory will be hosted as part of your documentation. ``$READTHEDOCS_OUTPUT`` is an :doc:`environment variable </reference/environment-variables>` that references the location where outputs should be stored.
```

Reply:
@humitos did we document `$READTHEDOCS_OUTPUT` anywhere yet?

Reply:
Nope, we opened this issue to track this work: https://github.com/readthedocs/readthedocs.org/issues/9894

Reply:
I'd want to have it documented at least in the Env Vars section, before we put it here.

Reply:
Yes, this little change can stand alone.

No one is assigned to #9894, but I don't mind doing it :+1:

Reply:
@ericholscher sounds good to me 👍🏼